api/public_api_v1.yaml

```diff
@@ -210,14 +209,22 @@ definitions:
         $ref: '#/definitions/TimeToLive'
       content:
         $ref: '#/definitions/MessageContent'
-      status:
-        $ref: '#/definitions/MessageStatus'
+      senderOrganizationId:
+        type: string
     required:
       - id
       - fiscal_code
-      - time_to_live
       - content
-      - status
+      - senderOrganizationId
+  MessageStatus:
+    type: object
+    properties:
+      message:
```

Comment:
why message status contains the whole message ?

Reply:
better rename it, this is actually the response to `GetMessage`